network-api/networkapi/wagtailpages/templates/fragments/most_voted_rating.html

```diff
@@ -0,0 +1,8 @@
+{% load i18n %}
+
+<span class="body-small">
+    {% with vote_data=product.votes.get_most_voted %} 
+    <a class="body-small" href="#creepiness-vote">{% trans "People voted" %}</a>: 
```

Comment:
Let’s keep the semicolon as part of the string, it needs to be localized (e.g. French adding a non-breaking space) and is important context. If we don’t want to keep it as part of the link, then we should switch to a `{% blocktrans %}` and include the link + the semicolon in the string

```suggestion
    <a class="body-small" href="#creepiness-vote">{% trans "People voted:" %}</a> 
```